apis/management.cattle.io/v3/machine_types.go

```diff
@@ -319,5 +319,6 @@ type CloudCredential struct {
 }
 
 type CloudCredentialSpec struct {
+	DisplayName string `json:"displayName"`
```

Comment:
Should this have norman:required?

Reply:
@prachidamle we didn't have validation on this before, so I am not sure if adding it now would break anything for people who automate things and don't add anything. what do you think?

Reply:
oh, if name is not required to create credential, we dont need displayName as required.